servers/rendering/renderer_rd/shaders/forward_mobile/scene_forward_mobile_inc.glsl

```diff
@@ -73,6 +73,8 @@ struct Lightmap {
 	mediump mat3 normal_xform;
 	vec3 pad;
 	float exposure_normalization;
+	vec2 light_texture_size;
+	vec2 pad2;
```

Comment:
```suggestion
	vec2 light_texture_size;
	float exposure_normalization;
	float pad;
```

This should come out of the padding that exists already so we aren't making the struct any bigger